claimedValueG1Jac.ScalarMultiplicationAffine undefined

C:\Users\user\go\pkg\mod\github.com\crate-crypto\go-kzg-4844@v1.0.0\internal\kzg\kzg_verify.go:73:20: claimedValueG1Jac.ScalarMultiplicationAffine undefined (type bls12381.G1Jac has no field or method ScalarMultiplicationAffine)

--- a/go.mod
+++ b/go.mod
@@ -3,7 +3,7 @@ module github.com/crate-crypto/go-kzg-4844
 go 1.20
 
 require (
-	github.com/consensys/gnark-crypto v0.12.1
+	github.com/consensys/gnark-crypto v0.13.0
 	github.com/stretchr/testify v1.8.2
 	golang.org/x/sync v0.1.0
 	gopkg.in/yaml.v2 v2.4.0
@@ -17,7 +17,7 @@ require (
 	github.com/mmcloughlin/addchain v0.4.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/rogpeppe/go-internal v1.12.0 // indirect
-	golang.org/x/sys v0.9.0 // indirect
+	golang.org/x/sys v0.15.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )

--- a/internal/kzg/kzg_verify.go
+++ b/internal/kzg/kzg_verify.go
@@ -70,7 +70,9 @@ func Verify(commitment *Commitment, proof *OpeningProof, openKey *OpeningKey) er
 	var claimedValueG1Jac bls12381.G1Jac
 	var claimedValueBigInt big.Int
 	proof.ClaimedValue.BigInt(&claimedValueBigInt)
-	claimedValueG1Jac.ScalarMultiplicationAffine(&openKey.GenG1, &claimedValueBigInt)
+	var GenG1Jac bls12381.G1Jac
+	GenG1Jac.FromAffine(&openKey.GenG1)
+	claimedValueG1Jac.ScalarMultiplication(&GenG1Jac, &claimedValueBigInt)
 
 	//  In the specs, this is denoted as `P_minus_y`
 	//

